packages/components/src/Toggle/LabelToggle/LabelToggle.component.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getTheme } from '../../theme';
+
+import css from './LabelToggle.scss';
+
+const theme = getTheme(css);
+
+/**
+ *
+ * @param {Array.<Object>} buttons - buttons that should be rendered
+ * @param {string} name - the name attribute for all inputs
+ * @param {string} checked - the checked value
+ * @param {function} onChange - callback that handle the state change
+ */
+function LabelToggle({ buttons, name, checked, onChange }) {
+	const handleChange = e => {
+		onChange(e.target.value);
+	};
+
+	return (
+		<div className={theme('radio-toolbar')}>
+			{buttons.map(button => (
+				<React.Fragment key={`radio${button.value}`}>
+					<input
+						id={`radio${button.value}`}
```

Comment:
⚠️ using this pattern to generate id could be dangerous since you can have the same value between two Toggle, so clicking on the second could impact the first. (Like in the story)

Reply:
Updated with uuid, thx